dp_check/dp_check.go

```diff
@@ -142,32 +235,103 @@ func main() {
 	if syscall.Getuid() != 0 {
 		infoLog.Println("Not running as root, some checks may fail.")
 	}
-	// Check IPv6
-	runCheck("IPv6 routes", func() error {
-		var out string
-		var err error
-		cmd := "ip -6 route show"
-		infoLog.Printf("Check IPv6 routes with:|%v|...\n", cmd)
-		if out, err = getOutput(cmd); err != nil {
+
+	// Check if dp_check is running on a VM
+	runCheck("Running on GCE VM", func() error {
```

Comment:
nit: also, if we're not running on GCP, can we skip the following subsequent `runCheck` 's ?

1) IPv6 addresses and routes
2) LB AAAA queries
3) SRV queries
?